### PR TITLE
feat(blog): add Introducing Spokestack Tray blog post

### PR DESCRIPTION
**Pull request by timmywil**
### PR Checklist

Please review the [guidelines for contributing](CONTRIBUTING.md) to this repository.

- [x] I am requesting to **pull a topic/feature/bugfix branch** (right side). In other words, not _master_.
- [x] I have run `npm test` against my changes and tests pass.
- [x] I have added or edited necessary documentation, or no docs changes are needed.

### Description

This adds Mike's new blog post.

### CODE REVIEW

**Comment by timmywil** [comment]
@space-pope just looking for a 2nd pair of eyes

File: content/blog/introducing-spokestack-tray/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: Introducing Spokestack Tray – a Turnkey Voice Interface for Mobile Apps
+date: '2020-09-21'
+description: 'Today we announce Spokestack Tray: a UI component for iOS, Android, and React Native making it easy to add Spokestack to any mobile app. Create your own Independent Voice Assistant!'
+tags: Engineering, Product
```
Comment (space-pope) [inline content/blog/introducing-spokestack-tray/index.md:5]: Not sure this is an engineering post, but 🤷 .

File: content/blog/introducing-spokestack-tray/index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+author: mike
+draft: false
+---
+
+Today we are excited to announce our latest developer feature: Spokestack Tray, a mobile library that lets developers easily add a voice interface to their existing app through a UI component that opens inside the application. Spokestack Tray incorporates our voice services, such as a wakeword, speech recognition, natural language processing, and custom text-to-speech voices, into one easy-to-add package along with any existing mobile application. We've built a demo, the Bartender mobile app, that uses Spokestack Tray.
```
Comment (space-pope) [inline content/blog/introducing-spokestack-tray/index.md:10]:
```suggestion
Today we are excited to announce our latest developer feature: Spokestack Tray, a mobile library that lets developers easily add a voice interface to their existing app through a UI component that opens inside the application. Spokestack Tray incorporates our voice services—wakeword, speech recognition, natural language processing, and custom text-to-speech voices—into one component that's easy to add to any mobile application. We've built a demo, the Bartender mobile app, that uses Spokestack Tray.
```

File: content/blog/introducing-spokestack-tray/index.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+Here’s a video of Spokestack Tray working in our demo Bartender app:
+
+`vimeo: [Spokestack Tray Demo](459905947)`
+
+Download the Bartender app and try Spokestack Tray on your own phone:
+
+iOS: [https://apps.apple.com/us/app/get-the-bartender/id1530425843](https://apps.apple.com/us/app/get-the-bartender/id1530425843)
```
Comment (space-pope) [inline content/blog/introducing-spokestack-tray/index.md:18]: I would put these in bullets and just make the platform name the link text, but meh.

File: content/blog/introducing-spokestack-tray/index.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+2. Check out the documentation for [React Native Tray](https://github.com/spokestack/react-native-spokestack-tray) or [iOS Tray](https://github.com/spokestack/spokestack-tray-ios)
+3. Follow a tutorial: [Spokestack Tray iOS Tutorial](/blog/integrating-spokestack-in-ios) or [Spokestack Tray React Native Tutorial](/blog/integrating-spokestack-in-react-native)
+4. Get help in our [Spokestack Tray Help](https://forum.spokestack.io/t/spokestack-tray/56) forum
+
+For folks who are new to building voice applications, we have a [video overview of voice concepts](https://www.youtube.com/watch?v=wbJ8fZh-iQw) as well as documentation of [voice developer concepts](/docs/Concepts) as they relate to Spokestack.
+
+### Customizing Spokestack Tray
+
+At Spokestack, we believe flexibility and customization are keys to building successful conversational interfaces. We have a Figma template for Spokestack that we also used for Bartender, and you can use it to customize your app. We will be adding more design resources and tools over the next few weeks. Additionally, we help you further customize your app with custom wakewords, TTS, and your own cloud or on-device NLU model.
```
Comment (space-pope) [inline content/blog/introducing-spokestack-tray/index.md]: I thought we were linking the Figma template here...or was that a separate template on the docs landing page?

File: content/blog/introducing-spokestack-tray/index.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+### Want a free wakeword, NLU training, and custom TTS?
+
+After you get a demo up and running, let us know if we can build a custom wakeword or TTS for your mobile app. We’ll hook up the first three developers who show us a working version of Spokestack Tray in their mobile app with a free wakeword, NLU training, and custom TTS. [Email us](mailto:hello@spokestack.io) your working demo and we’ll help you add these features to your app so you can publish it live.
```
Comment (space-pope) [inline content/blog/introducing-spokestack-tray/index.md:46]: "email us your demo is a little vague"—should we ask for a link to a repo, a beta invite, or something else?

File: content/blog/introducing-spokestack-tray/index.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+### What's next?
+
+Expect an update on design tools, the Android library, and additional tutorials soon. Follow us on [Twitter](http://www.twitter.com/spokestack) for all updates. Can't wait to see how people use Tray with their mobile apps.
```
Comment (space-pope) [inline content/blog/introducing-spokestack-tray/index.md]:
```suggestion
Expect an update on design tools, the Android Tray library, and additional tutorials soon. Follow us on [Twitter](http://www.twitter.com/spokestack) for all updates. Can't wait to see how people use Tray with their mobile apps.
```

**Review by space-pope — COMMENTED** [review]
Had a couple more comments/questions after seeing it in blog format.

File: content/blog/introducing-spokestack-tray/index.md (hunk 4 above)
Comment (miketatum) [reply content/blog/introducing-spokestack-tray/index.md]: Yes, we are...just waiting on Figma to approve our community request.

File: content/blog/introducing-spokestack-tray/index.md (hunk 5 above)
Comment (miketatum) [reply content/blog/introducing-spokestack-tray/index.md:46]: I was thinking we could ask for a link to their repo or to Testflight, etc. through email.

File: content/blog/introducing-spokestack-tray/index.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+At Spokestack, we believe flexibility and customization are keys to building successful conversational interfaces. We have a Figma template for Spokestack that we also used for Bartender, and you can use it to customize your app. We will be adding more design resources and tools over the next few weeks. Additionally, we help you further customize your app with custom wakewords, TTS, and your own cloud or on-device NLU model.
+
+### Want a free wakeword, NLU training, and custom TTS?
+
+After you get a demo up and running, let us know if we can build a custom wakeword or TTS for your mobile app. We’ll hook up the first three developers who show us a working version of Spokestack Tray in their mobile app with a free wakeword, NLU training, and custom TTS. [Email us](mailto:hello@spokestack.io) your working demo and we’ll help you add these features to your app so you can publish it live.
+
+### What's next?
+
+Expect an update on design tools, the Android Tray library, and additional tutorials soon. Follow us on [Twitter](http://www.twitter.com/spokestack) for all updates. Can't wait to see how people use Tray with their mobile apps.
```
Comment (timmywil) [inline content/blog/introducing-spokestack-tray/index.md]:
```suggestion
Expect an update on design tools, the Android Tray library, and additional tutorials soon. Follow us on [Twitter](https://www.twitter.com/spokestack) for all updates. Can't wait to see how people use Tray with their mobile apps.
```

File: content/blog/introducing-spokestack-tray/index.md (hunk 4 above)
Comment (timmywil) [inline content/blog/introducing-spokestack-tray/index.md]:
```suggestion
At Spokestack, we believe flexibility and customization are keys to building successful conversational interfaces. We will be adding more design resources and tools over the next few weeks. Additionally, we help you further customize your app with custom wakewords, TTS, and your own cloud or on-device NLU model.
```

**Pull request merged**